src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -929,6 +941,14 @@ public function __call($method, $parameters)
 		{
 			return $this->callScope($scope, $parameters);
 		}
+		elseif ( ! is_null($this->model['fillable']) && in_array($column = 'is_'.$method, $this->model['fillable']))
```

Comment:
@lukasgeiter That's probably overkill. I'm suggesting that change because it's more readable as well as being faster. Readability is important. :)